recipes/pylibmc/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "1.5.1" %}
+{% set gh_org = "lericson" %}
+{% set gh_repo = "pylibmc" %}
+
+package:
+  name: {{ gh_repo }}
+  version: {{ version }}
+
+source:
+  fn: {{ gh_repo }}-{{ version }}.tar.gz
+  url: https://github.com/{{ gh_org }}/{{ gh_repo }}/archive/{{ version }}.tar.gz
+  sha256: 18fa39885fce5464977d657185f55e491315b84686533db58054605165fd9b06
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - python
+    - libmemcached
+    - glib
+
+  run:
+    - python
+    - libmemcached
+
+test:
+  requires:
+    - python
+  imports:
+    - pylibmc
+    - _pylibmc
```

Comment:
What is the reason for importing this even though it has a `_` prefix? Is it an `.so` or something?